examples/performance/in-vr/components/stats-in-vr.js

```diff
@@ -0,0 +1,38 @@
+/* global AFRAME */
+AFRAME.registerComponent('stats-in-vr', {
+  schema: {
+    // Number of frames for moving average.
+    frames: {default: 30}
+  },
+
+  init: function () {
+    this.renderTimes = [];
+  },
+
+  tick: function (t, dt) {
+    var color;
+    var fps;
+    var renderTimes = this.renderTimes;
+
+    renderTimes.push(dt);
+    if (renderTimes.length > this.data.frames) { renderTimes.shift(); }
+
+    fps = renderTimes.length / (renderTimes.reduce(sum) / 1000);
+
+    color = 'green';
+    if (fps < 80) {
```

Comment:
These are moving averages, so if the last 45 frames gave an average of < 80, it's bad. These ranges are from when I was trying it out in VR and feeling what feels bad. When the average was below 80, it felt pretty bad. I only felt good if it was stable at 90. Even 89 indicated there was a frame drop or two somewhere.